test/python/common.py

```diff
@@ -368,5 +394,77 @@ def _is_ci_fork_pull_request():
     return False
 
 
-SKIP_ONLINE_TESTS = os.getenv('SKIP_ONLINE_TESTS', _is_ci_fork_pull_request())
-SKIP_SLOW_TESTS = os.getenv('SKIP_SLOW_TESTS', True) not in ['false', 'False', '-1']
+def get_test_options(option_var='QISKIT_TESTS'):
+    """
+    Reads option_var from env and returns a dict in which the test options are set
+
+    Args:
+        option_var (str): The env var to read. Default: 'QISKIT_TESTS'
+
+    Returns:
+        dict: A dictionary with the format {<option>: (bool)<activated>}.
+    """
+    defaults = {
```

Comment:
These defaults are contradictory, not passing any flag would result in an option set which is inconsistent since `skip_online` and `run_online` would be `False` for both. 

That said, notice we have three groups of tests: `offline`, `online`, `slow`. These can run or be skipped, which implies 8 possibilities to be controlled by 3 booleans at most. But `offline` tests are always run so 2 booleans, one set to run by default (`online`), the other to skip (`slow`).

I suggest removing `run_online` and `skip_slow`. This will simplify the implication map increasing readability and maintainability.

Reply:
e6a5cd8

Reply:
`run_online` and `skip_online` are not exactly opposite. `run_online` forces the use network against the real API. I agree that the way to have that options is by having credentials.